readme.md

```diff
@@ -2,11 +2,12 @@
 
 The Jira/Mattermost plugin documentation is currently being updated and relocated to a new location: https://mattermost.gitbook.io/jira-plugin/ - let us know your thoughts on the new format in the [Plugin: Jira Channel](https://community-daily.mattermost.com/core/channels/jira-plugin) on our Mattermost community!
 
-
```

Comment:
Nit pick: In other part of the documents a double newline is used before `#`. Maybe just stick with it.

Reply:
Ha!  I adopted a markdown linter a while back, but I'll add this back.. I'm 0/5 for enforcing Markdown syntax

Reply:
Me neither :shrug: